### ISSUE TITLE
Send out transactional emails to users 3 days after Webmaker install

### ISSUE THREAD

**Issue by secretrobotron** [mozilla/login.webmaker.org#365]
We need to send emails to users 3 days after they install and sign up to use Webmaker app.

@jbuck's advice was to use a [worker process](https://github.com/mozilla/webmaker-events-service/blob/master/processes/remind-attendees.js) here, in login.webmaker.org, to wake up each day (or hour?) and send messages through [hatchet](https://github.com/jbuck/hatchet) to any 3-day-old users.

@hpauljohnson and @kevzawacki will have content for the email which will be sent out through [sawmill via webmaker-mailroom](https://github.com/mozilla/webmaker-mailroom/issues/43).

cc @hpauljohnson @jbuck

**Comment by thisandagain** [comment]
I think I'd prefer to spin-up a new "scheduler" service that uses hatches to push jobs into the SQS queue. It seems odd to piggyback that functionality on-top of login. Thoughts @simonwex @jbuck ?

**Comment by jbuck** [comment]
Yeah, I think Bobby and I discussed this idea before we had that discussion. Lets go with the scheduler idea, it's better
